### PR TITLE
Update meta.yaml to 3.2.1 - Please review 

### PR DESCRIPTION
**Pull request by bobleesj**
Updated meta.yaml to version 3.2.1 with SHA value of fc46d222ec17431fddfc7046498ccc98af4a51f4e619cc09f6113c495b780a55

Screenshot 2024-09-04 at 8 52 56 AM

Double-checked `SHA256`

### CODE REVIEW

**Comment by bobleesj** [comment]
@conda-forge-admin please rerender

**Comment by bobleesj** [comment]
The tests are failing. I am looking into the problem.

**Comment by sbillinge** [comment]
possibly removing the `run_tests.py` from the `recipes` will allow this to pass ci.  I am not sure why this is failing and it would be good to get to the bottom of it.  It may be related to what @tieqiong is working on that pytest can't find installed tests.  But a workaround is just to remove this file for now.

**Comment by sbillinge** [comment]
Please remove it for now and we can merge this.  I think perhaps the solution is what @tieqiong found which is a pretty big fix

**Comment by bobleesj** [comment]
CI fails after removing `run_tests`.py but it gives a good error message:

```
export SRC_DIR=/home/conda/feedstock_root/build_artifacts/diffpy.structure_1725471966662/test_tmp
import: 'diffpy'
import: 'diffpy.structure'
import: 'diffpy.structure.parsers'
import: 'diffpy.structure.applications'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/diffpy.structure_1725471966662/test_tmp/run_test.py", line 11, in <module>
    import diffpy.structure.applications
ModuleNotFoundError: No module named 'diffpy.structure.applications'
WARNING: Tests failed for diffpy.structure-3.2.1-pyhd8ed1ab_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
Traceback (most recent call last):
```

I will look into `diffpy.structure.applications`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,9 +31,8 @@ test:
     - diffpy
     - diffpy.structure
     - diffpy.structure.parsers
-    - diffpy.structure.applications
+    - diffpy.structure.apps
```
Comment (bobleesj) [inline recipe/meta.yaml:34]: Based on 3.2.0 release

**Comment by bobleesj** [comment]
@sbillinge Please review and merge! I will test via both pip install and conda install.

**Pull request merged**